src/sentry/deletions/defaults/organization.py

```diff
@@ -53,6 +53,9 @@ def get_child_relations(self, instance):
             ProjectTransactionThreshold,
         )
         relations.extend([ModelRelation(m, {"organization_id": instance.id}) for m in model_list])
+        import logging
```

Comment:
Are these meant to be permanent?

Reply:
No. I'm getting totally different results in CI (tests fail) versus my local environment (tests pass).